data/ignition/files/usr/local/bin/start-assisted-service.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+HOST=$(ip -4 -j address | jq -r '[.[].addr_info]|flatten(1)|map(select(.scope=="global"))|.[0].local')
```

Comment:
```suggestion
HOST_IP="$(ip -j route get "$(ip -j route show default | jq -r ".[0].gateway")" | jq -r '.[0].prefsrc')"
```

Reply:
I realize that I forgot to suggest the change of HOST to HOST_IP in the sed command

Reply:
I don't think we should change it to HOST_IP because if it were an IPv6 address (in future) it must be surrounded by [] - it is actually the 'host' part of a URL.